src/test/java/io/jenkins/plugins/conventionalcommits/utils/CurrentVersionTest.java

```diff
@@ -156,7 +156,33 @@ public void testNpmProjectVersion() throws IOException, InterruptedException {
   }
 
   @Test
-  public void should_throw_npe_if_null_directory() {}
+  public void should_read_helm_chart_current_version() throws IOException, InterruptedException {
```

Comment:
I agree with @ADI10HERO, function name should use camelcase to follow the current code base code convention

Reply:
`should_read_helm_chart_current_version()` -> `shouldReadHelmChartCurrentVersion()`